modules/rational-evil.el

```diff
@@ -39,19 +34,20 @@
 ;; Set some variables that must be configured before loading the package
 (customize-set-variable 'evil-want-integration t)
 (customize-set-variable 'evil-want-keybinding nil)
+(customize-set-variable 'evil-want-C-i-jump t)
```

Comment:
The point of the `rational-evil-vim-muscle-memory` variable seemed to be to (by default) leave these at their default state, but if the user wished to turn them on, then these settings would apply. In this code, they are applied by default regardless rather than opted in via the `rational-evil-vim-muscle-memory` value.

Additionally, the `'evil-want-C-i-jump` value is set to `t` here and to `nil` a few lines below.

Reply:
i think that it depends on what rational wants as a whole, its opt in or out?

Reply:
I made it opt-in because, even though you are using evil, it doesn't mean that you want evil to overtake some Emacs keybinds. I think it should be a user choice if you want to prioritize vim muscle memory or using vim motion in combination with Emacs ones.

In my opinion the hole point of Rational Emacs is to stick to the defaults, or at least packages that take advantage of built-in behavior, and make some niceties, that go away from the default, opt-in.

I would even say to change `C-d` to be opt-in. But this is not something I can decide alone.

Reply:
Yep, I think opt-in is always a better choice.  Instead of being a "we choose everything for you" distro, this should be more of a "add things you might want quickly" distro.

Reply:
I also agree that `C-u` should be opt-in because it's a critical key binding for Emacs (`universal-argument`) and changing that will break standard Emacs usage patterns.